R/query.R

```diff
@@ -53,20 +56,33 @@ setMethod("dbFetch", "MariaDBResult",
 #' @export
 setMethod("dbSendQuery", c("MariaDBConnection", "character"),
   function(conn, statement, params = NULL, ...) {
-    statement <- enc2utf8(statement)
+    dbSend(conn, statement, params, FALSE)
+  }
+)
 
-    rs <- new("MariaDBResult",
-      sql = statement,
-      ptr = result_create(conn@ptr, statement)
-    )
+#' @rdname query
+#' @export
+setMethod("dbSendStatement", signature("MariaDBConnection", "character"),
+  function(conn, statement, params = NULL, ...) {
+    dbSend(conn, statement, params, TRUE)
```

Comment:
`is_statement = TRUE` instead of `TRUE` is a bit easier to understand. Same above for `FALSE`.